declarations/Autobazar.EU.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "Autobazar.EU",
+  "documents": {
+    "Ad Publishing Policy": {
```

Indeed, only section X (“Paid services of the Portal - publication of an offer/advertisement and additional services” according to automated translation) seems to fit that description. The overall document seems to mix terms for end users and for commercial partners.

Our jurisprudence has been to use the broader encompassing document type in case it is mixed like this:
```suggestion
    "Terms of Use": {
```

If we wanted to be more precise, we could indeed split per section, but that is significant additional work.